scripts/kind-e2e/e2e.sh

```diff
@@ -324,14 +332,94 @@ setup_custom_cni
 if [[ $3 = true ]]; then
     enable_logging
 fi
+
 install_helm
 if [[ $4 = true ]]; then
     enable_kubefed
 fi
 kind_import_images
 setup_broker
-setup_cluster2_gateway
-setup_cluster3_gateway
+# Verify SubM Broker secrets
+collect_subm_vars cluster1
+verify_subm_broker_secrets cluster1
+
+if [[ $5 = operator ]]; then
+    operator=true
+
+    for i in 2 3; do
+      # Create CRDs required as prerequisite submariner-engine
+      # TODO: Eventually OLM should handle this
+      create_subm_endpoints_crd cluster$i
```

Comment:
Okay, done in the latest version.

Reply:
I agree about the `context` variable. As far as the context goes, I’d rather have it re-specified on every invocation (as in the `e2e.sh` script) — otherwise it makes it more complicated to debug clusters: the scripts rely on the current `kubectl` state, which can be changed externally.